.github/workflows/publish_to_pypi.yaml

```diff
@@ -0,0 +1,25 @@
+name: pypi release
+on:
+  create:
+    tags:
+      - 'v*.*.*'
+jobs:
+  build_and_publish_to_pypi:
```

Comment:
Do you not need to create a matrix, to set up both python2 and python3?

Reply:
We only catering for python 3.